src/components/PublicationRequestFormSections/LinkAgreementForm/LinkAgreementForm.js

```diff
@@ -8,20 +8,20 @@ import {
   Col,
   Checkbox,
   TextField,
+  IconButton,
 } from '@folio/stripes/components';
 
 import { get } from 'lodash';
 import { Registry } from '@folio/handler-stripes-registry';
 
-
 const LinkAgreementForm = () => {
   const { initialValues, values } = useFormState();
   const { change } = useForm();
 
   const resourceReg = Registry.getResource('agreement');
   const LookupComponent = resourceReg?.getLookupComponent() ?? TextField;
 
-  const initialAgreement = (get(initialValues, 'agreement.remoteId_object'));
+  const initialAgreement = get(initialValues, 'agreement.remoteId_object');
```

Comment:
I know this is here from an older commit, but we really don't need lodash get here.

`initialValues?.agreement?.remoteId_object` will do the trick.

You would need it if you were passing in a variable path string, but this one is static so it's fine to use null coalescing